resteasy-spring-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -55,8 +55,17 @@
                     <groupId>org.wildfly</groupId>
                     <artifactId>wildfly-ee-galleon-pack</artifactId>
                 </exclusion>
+                <!-- todo: fix in RESTEasy side -->
+                <exclusion>
+                    <groupId>org.reactivestreams</groupId>
+                    <artifactId>reactive-streams</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.reactivestreams</groupId>
+            <artifactId>reactive-streams</artifactId>
+        </dependency>
```

Comment:
I'm not sure I follow this. The `org.jboss.resteasy:resteasy-client` has a dependency on this. If WildFly has a different version dependency, then we should be able to exclude it and not include it directly here unless Spring itself uses it.